go.mod

```diff
@@ -27,10 +27,12 @@ require (
 	go.uber.org/atomic v1.4.0 // indirect
 	golang.org/x/net v0.0.0-20190619014844-b5b0513f8c1b // indirect
 	golang.org/x/sync v0.0.0-20190423024810-112230192c58 // indirect
+	golang.org/x/sys v0.0.0-20190618155005-516e3c20635f // indirect
 	google.golang.org/grpc v1.20.1
-	gopkg.in/src-d/go-billy-siva.v4 v4.6.0
-	gopkg.in/src-d/go-billy.v4 v4.3.2
+	gopkg.in/src-d/go-billy-siva.v4 v4.5.1
+	gopkg.in/src-d/go-billy.v4 v4.3.0
```

Comment:
also do not revert go-billy to an older version please.

Reply:
Fixed.